tests/tools/oesign/sign-and-verify.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+import sys
+import subprocess
+import argparse
+
+if __name__ == "__main__":
+
+    arg_parser = argparse.ArgumentParser(description="Invokes the oesign sign command and attempts to load the signed enclave")
+    arg_parser.add_argument('--oesign-path', default=None, type=str, help="Path to the oesign tool")
+    arg_parser.add_argument('--oesign-args', default=None, type=str, help="Additional arguments to be passed to oesign")
+    arg_parser.add_argument('--enclave-path', default=None, type=str, help="Path to the enclave binary to be signed")
+    arg_parser.add_argument('--host-path', default=None, type=str, help="Path to the enclave host app used to launch the enclave")
+
+    args = arg_parser.parse_args()
+    print(args)
+
+    if not args.enclave_path or \
+       not args.host_path or \
+       not args.oesign_args or \
+       not args.oesign_path:
+
+        arg_parser.print_help()
+        sys.exit(1)
+
+    sign_cmd = args.oesign_path + ' sign --enclave ' + args.enclave_path + ' ' + args.oesign_args
```

Comment:
An idiomatic way to compose the arguments is to use `shlex.join`

Reply:
Thanks for the reminder to use Python idioms, this was carried over from the old scripts. Since the intent is to embed variables in a constant string structure instead of concatenating arbitrary tokens, I've switched to `.format()` pattern of use as generally recommended.